File: ContainsDuplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> skup = new HashSet<Integer>();
        for(int key : nums) {
            if(skup.contains(key)) {
                return true;
            }
            skup.add(key);
        }
        return false;
    }
}

File: Contains Duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> skup = new HashSet<int>();
        foreach(int key in nums) {
            if(skup.Contains(key)) {
                return true;
            }
            skup.Add(key);
        }
        return false;
    }
}
